src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForInitializerPadCheck.java

```diff
@@ -152,7 +152,7 @@ public void visitToken(DetailAST ast) {
             final int[] line = getLineCodePoints(lineIdx);
             final int before = ast.getColumnNo() - 1;
             // don't check if semi at beginning of line
-            if (!CodePointUtil.hasWhitespaceBefore(before, line)) {
+            if (ast.getColumnNo() > 0 && !CodePointUtil.hasWhitespaceBefore(before, line)) {
```

Comment:
Why not `if (before >= 0 &&` ?

For me it looks more clear.

Reply:
It can be both, but for current code pitest coverage is a little bit better.
Current code `>> Generated 2392 mutations Killed 2380 (99%)`
You suggestion `>> Generated 2391 mutations Killed 2378 (99%)`